linked_list/linked_list.py

```diff
@@ -3,33 +3,74 @@
 list node that holds a single value
 and a reference to the next node in the list
 """
+
+
 class Node:
-  def __init__(self, value=None, next_node=None):
-    self.value = value
-    self.next_node = next_node
+    def __init__(self, value=None, next_node=None):
+        self.value = value
+        self.next_node = next_node
 
-  def get_value(self):
-    return self.value
+    def get_value(self):
+        return self.value
 
-  def get_next(self):
-    return self.next_node
+    def get_next(self):
+        return self.next_node
+
+    def set_next(self, new_next):
+        self.next_node = new_next
 
-  def set_next(self, new_next):
-    self.next_node = new_next
 
 class LinkedList:
-  def __init__(self):
-    self.head = None
-    self.tail = None
+    def __init__(self):
+        self.head = None
+        self.tail = None
+
+    def add_to_tail(self, value):
+        node = Node(value)
+
+        if self.tail is not None:
```

Comment:
good work!, but for efficiency try typing a conditional where it checks if self.head isn't present. Great start, also try to use comments to explain your work